src/main/java/org/cascadebot/cascadebot/music/KaraokeHandler.java

```diff
@@ -0,0 +1,112 @@
+package org.cascadebot.cascadebot.music;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.TextChannel;
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.apache.commons.lang.StringEscapeUtils;
+import org.apache.tools.ant.filters.StringInputStream;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.cascadebot.cascadebot.MDCException;
+import org.cascadebot.cascadebot.data.language.Language;
+import org.cascadebot.cascadebot.messaging.MessageType;
+import org.cascadebot.cascadebot.messaging.Messaging;
+import org.jetbrains.annotations.NotNull;
+import org.w3c.dom.Document;
+import org.w3c.dom.NamedNodeMap;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class KaraokeHandler {
+
+    private static Map<Long, Boolean> karaokeEnabled = new HashMap<>();
+
+    public static boolean isKaraoke(Long guildId) {
+        if (karaokeEnabled.get(guildId) == null) {
+            return false;
+        } else { return karaokeEnabled.get(guildId); }
+    }
+
+    public static void setKaraoke(Long guildId, Boolean status) {
+        karaokeEnabled.put(guildId, status);
+    }
+
+    public static void getSongLyrics(String trackId, TextChannel channel, Long guildId, Message message) throws ParserConfigurationException {
+        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+        factory.setValidating(true);
+        factory.setIgnoringElementContentWhitespace(true);
+        DocumentBuilder builder = factory.newDocumentBuilder();
+        Request request = new Request.Builder().url("https://video.google.com/timedtext?lang=en&v=" + trackId).build();
+        CascadeBot.INS.getHttpClient().newCall(request).enqueue(new Callback() {
+            @Override
```

Comment:
```suggestion

            @Override
```

Reply:
Inconsistent spacing after overriding methods

Reply:
There should be a newline before the end and after the start of a class, whether it is anonymous or not

Reply:
Still viable